tests/test_shardformer/test_layer/test_flash_attention/test_flash_attention_for_opt.py

```diff
@@ -0,0 +1,34 @@
+import torch
+import torch.nn as nn
+from torch.testing import assert_close
+from transformers.models.opt.modeling_opt import OPTAttention
+
+from colossalai.shardformer.layer import FlashAttentionForOPT
+
+
+def test_flash_attention_for_opt():
+    BATCH, N_HEADS, N_CTX, D_HEAD = 4, 48, 4096, 64
+
+    # generate input
+    hidden_states = torch.randn((BATCH, N_CTX, N_HEADS * D_HEAD),
+                                dtype=torch.float32,
+                                device="cuda",
+                                requires_grad=True)
+    key_value_states = torch.randn((BATCH, N_CTX, N_HEADS * D_HEAD),
+                                   dtype=torch.float32,
+                                   device="cuda",
+                                   requires_grad=True)
+    attention_mask = torch.ones((BATCH, 1, N_CTX, N_CTX), dtype=torch.float32, device="cuda")
+
+    opt_attention = OPTAttention(embed_dim=D_HEAD * N_HEADS, num_heads=N_HEADS, dropout=0, is_decoder=True,
+                                 bias=True).to("cuda")
+    opt_flash_attention = FlashAttentionForOPT().from_native_module(opt_attention, None).to("cuda")
```

Comment:
This is not correct, `from_native_module` is a staticmethod, you cannot use on an object.

```python
FlashAttentionForOPT().from_native_module -> FlashAttentionForOPT.from_native_module
```